### ISSUE TITLE
VQE method 2 with qiskit raises exception

### ISSUE THREAD

**Issue by NosVladimir** [SRI-International/QC-App-Oriented-Benchmarks#428]
Hello, I was trying to run VQE method 2 with qiskit but it prompts the following exceptions:

```
Executing [17] circuits with num_qubits = 4
ERROR: failed to execute result_handler for circuit 4 ZIII
... exception = unsupported operand type(s) for *: 'dict' and 'float'
ERROR: failed to execute result_handler for circuit 4 XXII
... exception = unsupported operand type(s) for *: 'dict' and 'float'
ERROR: failed to execute result_handler for circuit 4 YYII
... exception = unsupported operand type(s) for *: 'dict' and 'float'
```
The same exception pops up for all the Pauli strings. VQE method 1 works fine.

**Comment by rtvuser1** [comment]
I believe this is due to the changes made to Qiskit earlier this year, where opflow was deprecated.  
This is a known issue and will be investigated at some point.
However, if you have a solution, please feel free to issue a PR with the change.  
Otherwise closure of the issue will be announced as a comment here once the issue is fixed.

**Comment by rtvuser1** [comment]
Issue closed with PR #576
